src/lib/containers/entity/page/CreatedByModifiedBy.tsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react'
+import {
+  Box,
+  Breadcrumbs,
+  Skeleton,
+  SxProps,
+  Tooltip,
+  Typography,
+} from '@mui/material'
+import {
+  BUNDLE_MASK_LAST_UPDATED_ON,
+  SMALL_USER_CARD,
+} from '../../../utils/SynapseConstants'
+import {
+  useGetEntity,
+  useGetQueryResultBundleWithAsyncStatus,
+} from '../../../utils/hooks/SynapseAPI'
+import {
+  convertToEntityType,
+  entityTypeToFriendlyName,
+  isDataset,
+  isTable,
+} from '../../../utils/functions/EntityTypeUtils'
+import UserCard from '../../UserCard'
+import { formatDate } from '../../../utils/functions/DateFormatter'
+import ConditionalWrapper from '../../utils/ConditionalWrapper'
+import dayjs from 'dayjs'
+import { InfoTwoTone } from '@mui/icons-material'
+
+export type CreatedByModifiedByProps = {
+  entityId: string
+  versionNumber?: number
+}
+
+function Separator() {
+  return (
+    <Typography variant={'breadcrumb1'} sx={{ color: 'grey.700' }}>
+      /
+    </Typography>
+  )
+}
+
+export function CreatedByModifiedBy(props: CreatedByModifiedByProps) {
+  const { entityId, versionNumber } = props
+  const entityIdWithVersion = `${entityId}${
+    versionNumber ? `.${versionNumber}` : ''
+  }`
+
+  const { data: entity } = useGetEntity(entityId, versionNumber)
+
+  const { data: tableQueryResult } = useGetQueryResultBundleWithAsyncStatus(
+    {
+      entityId,
+      query: {
+        sql: `SELECT * FROM ${entityIdWithVersion} LIMIT 0`,
+      },
+      partMask: BUNDLE_MASK_LAST_UPDATED_ON,
+      concreteType: 'org.sagebionetworks.repo.model.table.QueryBundleRequest',
+    },
+    { enabled: !!(entity && isTable(entity)) },
+  )
```

Comment:
Just requesting the `lastUpdatedOn` so this additional query bundle request should have little additional impact. I will also be removing this part from the mask in SWC, so it won't be shown with the table.